npm-scripts.js

```diff
@@ -276,6 +298,43 @@ function lintWorker()
 	executeCmd(`${MAKE} lint -C worker`);
 }
 
+function flatcNode(clean = false)
+{
+	console.log('npm-scripts.js [INFO] flatcNode()');
+
+	// Build flatc if needed.
+	executeCmd(`${MAKE} -C worker flatc-all`);
+
+	const extension = isWindows ? '.exe' : '';
+	const flatc = path.resolve(path.join(
+		'worker', 'out', 'Release', 'build', 'subprojects', `flatbuffers-${FLATBUFFERS_VERSION}`, `flatc${extension}`));
```

Comment:
Here we are hardcoding `Release` path which makes it no longer possible to build `Debug`. 

Take a look to `Worker.ts`:

```ts
// If env MEDIASOUP_WORKER_BIN is given, use it as worker binary.
// Otherwise if env MEDIASOUP_BUILDTYPE is 'Debug' use the Debug binary.
// Otherwise use the Release binary.
const workerBin = process.env.MEDIASOUP_WORKER_BIN
	? process.env.MEDIASOUP_WORKER_BIN
	: process.env.MEDIASOUP_BUILDTYPE === 'Debug'
		? path.join(__dirname, '..', '..', 'worker', 'out', 'Debug', 'mediasoup-worker')
		: path.join(__dirname, '..', '..', 'worker', 'out', 'Release', 'mediasoup-worker');
```

Reply:
good point

Reply:
handled here [227a8a0](https://github.com/versatica/mediasoup/pull/927/commits/227a8a0adde62e2f594e914122318060cd43089e)